src/app/core/user/user.ts

```diff
@@ -34,7 +34,23 @@ export class User extends Entity {
     label: $localize`:Label of username:Username`,
     validators: { required: true },
   })
-  name: string;
+  set name(value: string) {
```

Comment:
let's maybe change the property to `username` (or will this break dozens of things?). For some clients we introduced additional firstname and lastname properties to be used in the UI. "username" would make the purpose of this one more explicit

Reply:
At the moment it would break a dozens of things (all the current user entities) So lets get this out and if we ever want to migrate this, we can do this independently.